### DISCUSSION TITLE
How can we make checkboxes required?

### DISCUSSION THREAD

**Discussion by matt-aircritical** [docusealco/docuseal#117] [Q&A]
First of all, thank you so much for creating this software [AlexandrToorchyn](https://github.com/AlexandrToorchyn) I can not believe how well the software is put together and how exciting it is to see Vue and ruby and all the other technologies involved in one application. This software will give me a reason to really explore these technologies and see how they fit together.

I greatly appreciate the jump in technology this software is over what we have had previously in regards to PDF signing, storage, logs the entire package in an open source project, it's just mind blowing. Everything I need is here except one thing:

Is there a way to make a _**checkbox required?**_ 

I am studying the code but have limited Vue and ruby experience to try and see what I am missing about this but am excited to contribute to this project in the future.

Thank you so much for such great software and making it open source.

**Comment by AlexBTurchyn** [comment]
@matt-aircritical We added the ability to make checkboxes required in the latest release (1.2.2)
